### PR TITLE
Commando Update

### PR DESCRIPTION
**Pull request by ghostiek**
- Changed command module to use Commando
- Added a cat and dog command
- Added a basic ping command

### CODE REVIEW

File: src/commands/Cute/cat.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		super(client, {
+			name: 'cat',
+			group: groupName,
+			memberName: 'cat',
+			description: 'Posts a cat picture.',
+		});
+	}
+
+	async run(message) {
+        const content = await request.get(url);
```
Comment (antoinegag) [inline src/commands/Cute/cat.js]: Why not use the `fetch` API?

**Comment by ghostiek** [comment]
@antoinegag I think I fixed the indentation problems.

As for the fetch lib we can use whatever we want, I just used one of the first things I found as I don't know much about the ideal js libs to use

**Comment by ghostiek** [comment]
Also on another note, the code still works but eslint was yelling at me for this
https://eslint.org/docs/rules/class-methods-use-this

in each of the command files. Essentially I can't make the run function static because Commando will simply not recognize it and I don't find it elegant to have to add in a comment in every command file. Is there a way to edit eslint preferences to make it stop checking for class methods to use this in the commands directory?

**Comment by antoinegag** [comment]
> @antoinegag I think I fixed the indentation problems.
> 
> As for the fetch lib we can use whatever we want, I just used one of the first things I found as I don't know much about the ideal js libs to use

I'd use that https://www.npmjs.com/package/node-fetch

File: package.json [hunk 2]
```diff
@@ -11,7 +11,10 @@
   },
   "license": "MIT",
   "dependencies": {
-    "discord.js": "^11.4.2"
+    "discord.js": "^11.4.2",
+    "discord.js-commando": "^0.10.0",
+    "node-fetch": "^2.3.0",
+    "requests": "^0.2.2"
```
Comment (antoinegag) [inline package.json]: Can be safely removed

Comment (ghostiek) [reply package.json]: dont think so but npm uninstall requests didn't remove it

File: src/commands/Cute/cat.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const { Command } = require('discord.js-commando');
+const path = require('path');
+const fetch = require('node-fetch');
+
+const groupName = __dirname
```
Comment (antoinegag) [inline src/commands/Cute/cat.js]: You do this for each comment, consider making a function that you can reuse instead

File: .prettierrc [hunk 4]
```diff
@@ -6,5 +6,5 @@
   "trailingComma": "es5",
   "bracketSpacing": true,
   "arrowParens": "avoid",
-  "endOfLine": "lf"
+  "endOfLine": "auto"
```
Comment (CMCarlsen) [inline .prettierrc]: I'm actually curious why this was changed?

I specifically made this Line-Feed/lf on purpose, as I don't want crlf polluting the code base? Would prefer to keep this out unless there's a specific reason for switching it?

**Pull request merged**